site/src/components/Markdown/Markdown.tsx

```diff
@@ -0,0 +1,93 @@
+import Link from "@material-ui/core/Link"
+import { Theme, useTheme } from "@material-ui/core/styles"
+import Table from "@material-ui/core/Table"
+import TableBody from "@material-ui/core/TableBody"
+import TableCell from "@material-ui/core/TableCell"
+import TableContainer from "@material-ui/core/TableContainer"
+import TableHead from "@material-ui/core/TableHead"
+import TableRow from "@material-ui/core/TableRow"
+import { FC } from "react"
+import ReactMarkdown from "react-markdown"
+import SyntaxHighlighter from "react-syntax-highlighter"
+import { dracula as dark } from "react-syntax-highlighter/dist/esm/styles/hljs"
+import gfm from "remark-gfm"
+
+export interface MarkdownProps {
+  children: string
+}
+
+export const Markdown: FC<{ children: string }> = ({ children }) => {
+  const theme: Theme = useTheme()
+
+  return (
+    <ReactMarkdown
+      remarkPlugins={[gfm]}
+      components={{
+        a: ({ href, target, children }) => (
+          <Link href={href} target={target}>
+            {children}
+          </Link>
+        ),
+
+        // Adding node so the ...props don't have it
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
❓ help me understand this (sorry if it's obvious 😂) we destructure `node` but don't use it, how come?

does prefixing with `_` fix the ESLint warning?

Reply:
> we destructure node but don't use it, how come?

Because we are getting the `...props` and we don't want props having `node`. 

> does prefixing with _ fix the ESLint warning?

Yes, but if I use `_node` it will return `_node not found` because `_node` is not an attribute. I could do `node: _node` but I think it is more confusing than letting it as it is and adding an ignoring comment.

Reply:
Ahhh okay, that makes sense! Thanks for explaining!